plotting.py

```diff
@@ -0,0 +1,69 @@
+import pandas as pd
+import numpy as np
+import glob
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+def augment_csv_with_speed(df):
+    df['speed'] = df.apply(lambda row: np.linalg.norm([row.vx,row.vy,row.vz]),axis=1)
+
+def augment_csv_with_distance(df):
+    cx = df['x'].mean()
+    cy = df['y'].mean()
+    cz = df['z'].mean()
+    df['dist'] = df.apply(lambda row: np.linalg.norm([row.x - cx, row.y - cy, row.z -cz]), axis=1) 
+
+def find_repeat_time_idx(df):
+    drop_idx = {}
+    for id in df:
+        cur = float('inf')
+        for i,speed in zip(df.index,df[id]):
+            if cur == speed:
+                drop_idx[id] = i
+                break
+            else:
+                cur = speed
+    return drop_idx
+
+def drop_repeats(df, drop_idx):
+    for id,idx in drop_idx.items():
+        idx = list(range(idx,19))
+        df.loc[idx,id] = np.nan
+
+
+files = sorted(glob.glob('splitData/ids_c_*.csv'))
```

Comment:
you may want to change this to 'csvs/splitData/ids_c_*.csv' because I sorted the csv files into that directory